reportsrv/reportsrv.go

```diff
@@ -74,6 +79,7 @@ func (s *Server) verifyRequest(next http.Handler) http.Handler {
 
 func (s *Server) RegisterHandlers(r chi.Router) {
 	r.Use(s.verifyRequest)
+	r.Get("/report/{analysis_id}", s.serveReport)
```

Comment:
Noted above, but make this redirect to the slash-suffixed version, it breaks relative links otherwise

Reply:
Done.